plugins/CustomDimensions/vue/src/Edit/Edit.vue

```diff
@@ -0,0 +1,350 @@
+          } as unknown as CustomDimension;
+          this.addExtraction();
+        }
+      });
+    },
+    removeExtraction(index: number) {
+      if (index > -1) {
+        this.dimension.extractions.splice(index, 1);
+      }
+    },
+    addExtraction() {
+      if (this.doesScopeSupportExtraction) {
+        this.dimension.extractions.push({
+          dimension: 'url',
+          pattern: '',
+        });
+      }
+    },
+    createCustomDimension() {
+      this.isUpdatingDim = true;
+      CustomDimensionsStore.createOrUpdateDimension(
+        this.dimension,
+        'CustomDimensions.configureNewCustomDimension',
+      ).then(() => {
+        this.showNotification(translate('CustomDimensions_DimensionCreated'), 'success');
+        console.log('before reload');
+        CustomDimensionsStore.reload();
+        MatomoUrl.updateHashToUrl('/list');
+      }).finally(() => {
+        this.isUpdatingDim = false;
+      });
+    },
+    updateCustomDimension() {
+      this.isUpdatingDim = true;
+      CustomDimensionsStore.createOrUpdateDimension(
+        this.dimension,
+        'CustomDimensions.configureExistingCustomDimension',
+      ).then(() => {
+        this.showNotification(translate('CustomDimensions_DimensionUpdated'), 'success');
+        CustomDimensionsStore.reload();
+        MatomoUrl.updateHashToUrl('/list');
+      }).finally(() => {
+        this.isUpdatingDim = false;
+      });
+    },
+  },
+  computed: {
+    isLoading() {
+      return CustomDimensionsStore.isLoading.value;
+    },
+    isUpdating() {
+      return CustomDimensionsStore.isUpdating.value || this.isUpdatingDim;
+    },
+    create() {
+      return this.dimensionId === 0;
+    },
+    edit() {
+      return !this.create;
+    },
+    extractionDimensionsOptions() {
+      return CustomDimensionsStore.extractionDimensionsOptions.value;
+    },
+    availableScopes() {
+      return CustomDimensionsStore.availableScopes.value;
+    },
+    doesScopeSupportExtraction() {
+      if (!this.dimension?.scope || !this.availableScopes) {
+        return false;
+      }
+
+      const dimensionScope = this.availableScopes.find(
+        (scope) => scope.value === this.dimension.scope,
+      );
+      return dimensionScope?.supportsExtractions;
+    },
+    contentTitleText() {
+      return translate(
+        'CustomDimensions_ConfigureDimension',
+        ucfirst(this.dimensionScope),
+        `${this.dimension?.index || ''}`,
+      );
+    },
+    howToTrackManuallyText() {
+      const link = 'https://developer.piwik.org/guides/tracking-javascript-guide#custom-dimensions';
+      return translate(
+        'CustomDimensions_HowToTrackManuallyViaJsDetails',
+        `<a target=_blank href="${link}" rel="noreferrer noopener">`,
+        '</a>',
+      );
+    },
+    manuallyTrackCode() {
+      const exampleValue = translate('CustomDimensions_ExampleValue');
+      return `&dimension${this.dimension.idcustomdimension}=${exampleValue}}`;
```

Comment:
Seems there is a bracket too much.
```suggestion
      return `&dimension${this.dimension.idcustomdimension}=${exampleValue}`;
```